avocado/utils/partition.py

```diff
@@ -254,7 +254,7 @@ def _unmount_force(self, mountpoint):
         # Try to kill all pids
         for pid in (line.split()[1] for line in out.splitlines()[1:]):
             try:
-                process.system("kill -9 %s" % pid, ignore_status=True,
+                process.system("kill -9 %s" % pid.decode(), ignore_status=True,
```

Comment:
Based on the previous commits, shouldn't this respect the preferred system encoding? I mean, not Python's one?

Reply:
yes, I need to learn to use that... (old habits die hard)